bin/functions/workload-functions.sh

```diff
@@ -308,6 +309,17 @@ function ensure-mahout-release (){
     export_withlog HADOOP_CONF_DIR    
 }
 
+function ensure-tpcds-kit-ready (){
+    if [ ! -e "${DSDGEN_DIR}/dsdgen" ]; then
+        mv "${DSDGEN_DIR}/Makefile.suite" "${DSDGEN_DIR}/makefile"
+        make -C ${DSDGEN_DIR}
```

Comment:
This requires some prerequisites to compile like `gcc make flex bison byacc`.  We need at least add these in the doc?